### PR TITLE
Fix: Use enum values when dumping models

### PR DESCRIPTION
**Pull request by raminqaf**

### CODE REVIEW

File: tests/pipeline/resources/streams-bootstrap/defaults.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 streams-bootstrap:
   version: "3.0.3"
   values:
+    imagePullPolicy: IfNotPresent
```
Comment (disrupted) [inline tests/pipeline/resources/streams-bootstrap/defaults.yaml:4]: are you sure about this change? this deviates from the default defined in streams-bootstrap.

**Review by disrupted — COMMENTED** [review]
could you briefly explain what the bug was?

File: tests/pipeline/resources/streams-bootstrap/defaults.yaml (hunk 1 above)
Comment (raminqaf) [reply tests/pipeline/resources/streams-bootstrap/defaults.yaml:4]: I added the imagePullPolicy here only for testing. The problem was that when we called `kpops manifest ...` the values of the enums were not dumped as String. Here is the error:
```
yaml.representer.RepresenterError: ('cannot represent an object', <ImagePullPolicy.IF_NOT_PRESENT: 'IfNotPresent'>)
```

Comment (disrupted) [reply tests/pipeline/resources/streams-bootstrap/defaults.yaml:4]: gotcha! if you revert the change I'll approve :)

**Review by disrupted — APPROVED** [review]

**Pull request merged**